src/main/java/core/basesyntax/Pair.java

```diff
@@ -1,5 +1,47 @@
 package core.basesyntax;
 
-class Pair {
+class Pair<T, K> {
+    private T number;
```

Comment:
it's not always will be number and string, so this naming could be confusing, try to use more general names

Reply:
Done